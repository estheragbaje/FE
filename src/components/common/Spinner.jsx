import React from 'react';
import { Triple } from 'react-preloading-component';
import styled from 'styled-components';

const StyledSpinner = styled.div`
  margin: 10rem 0;
`;

export default function Spinner() {
  return (
    <StyledSpinner>
      <Triple color="#63beff" size={80} />
    </StyledSpinner>
  );
}
