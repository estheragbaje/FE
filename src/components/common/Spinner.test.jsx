import React from 'react';
import { shallow } from 'enzyme';

import Spinner from './Spinner';

describe('<Spinner />', () => {
  it('renders the Spinner component correctly', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
