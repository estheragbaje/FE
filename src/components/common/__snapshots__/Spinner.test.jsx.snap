// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> renders the Spinner component correctly 1`] = `
<styled.div>
  <n
    color="#63beff"
    size={80}
  />
</styled.div>
`;
